FWCore/Framework/interface/EventProcessor.h

```diff
@@ -369,7 +369,7 @@ namespace edm {
 
     bool printDependencies_ = false;
     bool deleteNonConsumedUnscheduledModules_ = true;
-    bool firstItemAfterLumiMerge_ = true;
+    bool needToCallNext_ = true;
```

Comment:
I'm really not a fan of all the `naked` calls to look at or change the value of `needToCallNext_` in the `EventProcessor.cc` file. Could such behavior be moved to new member functions?

Reply:
Done. Thanks.